File: reverse.py
#a program that reverses a number
def reverse(number):
    reverse = 0
    while number > 0:
        reverse = reverse * 10 + number % 10
        number = number // 10
    return reverse
def main():
    number = int(input("Enter a number: "))
    print("The reverse of", number, "is", reverse(number))
main()

File: reverse.java
import java.util.Scanner;

/**
 * reverse
 */
public class reverse {

    public static void main(String[] args) {
        //a program to reverse a number
        int num; 
        Scanner entry = new Scanner(System.in);   
        System.out.println("Enter a number: ");
        num = entry.nextInt();
        int reverse = 0;
        while(num != 0) {
            reverse = reverse * 10;
            reverse = reverse + num % 10;
            num = num / 10;
        }
        System.out.println("Reverse of the number is: " + reverse);
    }
}